packages/autocomplete-theme-classic/src/theme.scss

```diff
@@ -161,110 +211,140 @@ body {
-    padding-bottom: var(--aa-spacing-half);
-    padding-top: var(--aa-spacing-half);
+    overflow-y: auto;
+    padding: 0;
+    padding: var(--aa-spacing-half) var(--aa-spacing) var(--aa-spacing)
+      var(--aa-spacing-half);
     position: relative;
     text-align: left;
 
-    .aa-PanelLayoutResults {
-      overflow-x: hidden;
-      overflow-y: auto;
-      width: 50%;
+    &.aa-PanelLayout-cols-two--golden {
+      display: grid;
+      grid-template-columns: 39.2% auto;
+      overflow: hidden;
+      padding: 0;
     }
-    .aa-PanelLayoutPreview {
-      border-left: solid 1px var(--aa-selected-color);
-      flex-shrink: 1;
-      max-width: 50%;
+    &.aa-PanelLayout-cols-two {
+      display: grid;
+      grid-template-columns: repeat(2, minmax(0, 1fr));
       overflow: hidden;
+      padding: 0;
+    }
+    &.aa-PanelLayout-cols-three {
+      display: grid;
+      grid-template-columns: repeat(3, minmax(0, 1fr));
+      overflow: hidden;
+      padding: 0;
+    }
+
+    .aa-Panel--Scrollable {
+      margin: 0;
+      max-height: var(--aa-panel-max-height);
+      overflow-x: hidden;
+      overflow-y: auto;
+      padding: var(--aa-spacing-half) var(--aa-spacing) var(--aa-spacing) 0;
+      &:first-child {
+        margin-left: var(--aa-spacing-half);
+      }
     }
   }
-  .aa-Panel--Scrollable {
-    overflow-y: auto;
+  // reset
+  * {
+    @extend %reset;
+  }
+  button {
+    appearance: none;
+    background: none;
+    border: 0;
+    margin: 0;
+    padding: 0;
   }
-
   // when a query isn't resolved yet
   &.aa-Panel--stalled {
-    filter: grayscale(1);
-    opacity: 0.8;
-    @media screen and (prefers-reduced-motion: reduce) {
-      transition: opacity 200ms ease-in;
+    .aa-Source {
+      filter: grayscale(1);
+      opacity: 0.8;
     }
   }
 }
 
 // ----------------
-// Sources
+// 5. Sources
 // Each source can be styled independently
 // ----------------
 .aa-Source {
+  margin: 0;
+  padding: 0;
   position: relative;
   width: 100%;
+  @at-root .aa-SourceNoResults {
+    font-size: 1em;
+    margin: 0;
+    padding: var(--aa-spacing);
+  }
+  // list of results inside the source
+  @at-root .aa-List {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+    position: relative;
+  }
   // source title
   .aa-SourceHeader {
-    line-height: var(--aa-spacing);
-    margin: var(--aa-spacing-half) calc(var(--aa-spacing-half) + 2px);
+    margin: var(--aa-spacing-half) 0.5em var(--aa-spacing-half) 0;
+    padding: 0;
     position: relative;
     // Title typography
-    .aa-SourceHeaderTitle {
+    @at-root .aa-SourceHeaderTitle {
       background: var(--aa-background-color);
       color: var(--aa-primary-color);
       display: inline-block;
       font-size: 0.8em;
       font-weight: 600;
-      padding-right: var(--aa-spacing-half);
+      margin: 0;
+      padding: 0 var(--aa-spacing-half) 0 0;
       position: relative;
       text-transform: capitalize;
```

Comment:
Do we really need to capitalize? If you put a longer sentence in there it will look weird.

Since users provide the header template, it's probably best to give them control.

Reply:
No—we forgot to remove this indeed.

Reply:
See cd6facdb860e976d201f089ea9827ea411787472.